src/router/index.ts

```diff
@@ -14,4 +18,19 @@ const router = new VueRouter({
   }
 })
 
+router.beforeEach((to, from, next) => {
+  // If the route requires authentication
+  if (to.matched.some(record => record.meta.requiresAuth)) {
+    // If we have a token dont have a token
```

Comment:
This comment needs correction ..

Reply:
Please add "FUTURE" comment indicating that we should handle token expiry at some point.

Reply:
Resolved in latest commit